pkgs/development/python-modules/pyopencl/default.nix

```diff
@@ -1,54 +1,58 @@
 {
   lib,
   stdenv,
-  fetchFromGitHub,
+  darwin,
+  ocl-icd,
   buildPythonPackage,
+  fetchFromGitHub,
 
   # build-system
   cmake,
-  scikit-build-core,
-  pathspec,
-  ninja,
   nanobind,
+  ninja,
+  numpy,
+  oldest-supported-numpy,
```

Comment:
Can be removed? I'd recommend using `nixd` to detect unused inputs.

Reply:
or statix